tests/unit/test_runner.py

```diff
@@ -9,7 +9,13 @@
 from jbi.environment import get_settings
 from jbi.errors import ActionNotFoundError, IgnoreInvalidRequestError
 from jbi.models import ActionContext
-from jbi.runner import Executor, execute_action, lookup_action
+from jbi.queue import DeadLetterQueue
+from jbi.runner import Actions, Executor, execute_action, execute_or_queue, lookup_action
+
+
+@pytest.fixture
+def mock_queue():
+    return mock.MagicMock(spec=DeadLetterQueue)
```

Comment:
This could probably go to conftest if it's used across different modules

Reply:
Moving it to conftest now that the other PR has been merged.